android/src/main/java/tv/vizbee/rnsender/VizbeeCastButtonView.java

```diff
@@ -31,6 +31,28 @@ public void setTintColor(String tintColor){
         }
     }
 
+     public void setEnabled(boolean clickable) {
```

Comment:
Why does it have to be exposed to JS? Can't it be by default disabled as we are exporting only VizbeeCastButtonPressable?

Reply:
To support backward compatibility we can disable it in  native side